### PR TITLE
feat(cli): Print 'no resource msg' when `argo list/delete` returns zero workflows

### PR DESCRIPTION
**Pull request by gosoon**
Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this is a chore.
* [ ] The title of the PR is (a) [conventional](https://www.conventionalcommits.org/en/v1.0.0/), (b) states what changed, and (c) suffixes the related issues number. E.g. `"fix(controller): Updates such and such. Fixes #1234"`.  
* [ ] I've signed the CLA.
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My builds are green. Try syncing with master if they are not. 
* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: cmd/argo/commands/delete.go [hunk 1]
```diff
@@ -50,6 +50,12 @@ func NewDeleteCommand() *cobra.Command {
 				errors.CheckError(err)
 				workflows = append(workflows, listed...)
 			}
+
+			if len(workflows) == 0 {
+				fmt.Printf("No resources found\n")
+				return
+			}
```
Comment (simster7) [inline cmd/argo/commands/delete.go:57]: We should add this to other commands such as `lint`, `stop`, `resume`, etc. or check that it's already there. Can you do this for us?

Comment (alexec) [reply cmd/argo/commands/delete.go:57]: and we should add tests

Comment (gosoon) [reply cmd/argo/commands/delete.go:57]: ok, The “listWorkflows” method is called in the code and need to check if workflows is empty，I find the “list” and “delete” commands have the calls.

Comment (alexec) [reply cmd/argo/commands/delete.go:57]: What does kubectl do when this happens? We should try to be consistent please.

Comment (gosoon) [reply cmd/argo/commands/delete.go:57]: Kubectl is the same output:
```
$ kubectl delete pod -l name=myLabel
No resources found

$ kubectl get pod -n test
No resources found in test namespace.
```

File: cmd/argo/commands/delete.go [hunk 2]
```diff
@@ -50,6 +50,12 @@ func NewDeleteCommand() *cobra.Command {
 				errors.CheckError(err)
 				workflows = append(workflows, listed...)
 			}
+
+			if len(workflows) == 0 {
+				fmt.Printf("No resources found\n")
```
Comment (alexec) [inline cmd/argo/commands/delete.go:55]: How about "no workflows to delete?"

File: cmd/argo/commands/list.go [hunk 3]
```diff
@@ -53,6 +54,11 @@ func NewListCommand() *cobra.Command {
 			}
 			workflows, err := listWorkflows(ctx, serviceClient, listArgs)
 			errors.CheckError(err)
+			if len(workflows) == 0 {
+				fmt.Printf("No resources found\n")
```
Comment (alexec) [inline cmd/argo/commands/list.go:58]: "no workflows found"?

Comment (alexec) [reply cmd/argo/commands/list.go:58]: move into "PrintWorkflows" and add test?

**Review by alexec — CHANGES_REQUESTED** [review]
Changes requested

File: cmd/argo/commands/list.go (hunk 3 above)
Comment (gosoon) [reply cmd/argo/commands/list.go]: ok

**Review by alexec — APPROVED** [review]
LGTM!

**Pull request merged**